tests/test_github.py

```diff
@@ -71,6 +71,11 @@
 GITHUB_ORGS_URL = GITHUB_API_URL + "/users/zhquan_example/orgs"
 GITHUB_COMMAND_URL = GITHUB_API_URL + "/command"
 
+GITHUB_PULL_REQUEST_1_COMMIT_COMMENTS = GITHUB_REPO_URL + \
```

Comment:
@valeriocos 
Also, one more thing which I needed to ask is since the mocked commits in `github_request_pull_request_1_commits` and `github_request_pull_request_2_commits` data are both the same ('53b970ee04bbc435842c14a2cbfdd623faf74a65'), shall I define the same url twice as done here, or is it good to define it only once `GITHUB_PULL_REQUEST_COMMIT_COMMENTS`( no `_1_` or `_2_`) ?
Another solution is to change the mocked commit in one of the pull_request to say '53b970ee04bbc435842c14a2cbfdd623faf74a6**6**', and then define these two seperate urls.

Reply:
I would go for your `Another solution`, thus (if I understood it correctly) we should have two test files, one file including the comments for a commit in PR 1 and another file including the comments for a commit in PR 2 (as done the test data of pull request comments or commits).  What do you think ?

Reply:
Yes @valericos, this seemed better option to me!
The content for those two files would exactly be same, as I understand.

Reply:
Probably it's better to modify one of them. For instance, you can have one file with 4 comments and the other one with 2/3. The assertions in the tests could check the number of comments.
What do you think ?

Reply:
Okay, @valeriocos, would update it!